qt5/applet/qtquick-plugin/CMakeLists.txt

```diff
@@ -0,0 +1,56 @@
+set(CMAKE_AUTOMOC ON)
+find_package(Qt5 REQUIRED COMPONENTS Quick Core Qml)
+
+add_library(
+    uim-qtquick-plugin
```

Comment:
Okay, so there are two parts to a modern Kde/Plasma Applet:
1. __QtQuick (QML + JavaScript):__ This is the new Ui framework by the Qt library. It introduces a new language called QML. This is what the Plasma team wants everyone to write applets in.
1. **Native Plugin (C++)**: For things like this that need to interact with existing C++ code, they allow an extension mechanism called QtQuick Native Plugins (or QML Plugins). These get loaded automatically by the QML runtime if they're located in a particular system directory.

This "library" is just that native plugin. Since it's automatically by the runtime engine by its path, it's super important that it be in that specific directory. However, the name of the lib itself (the `.so` file) doesn't matter at all. I picked this name because I thought it would make it easier to identify in the source code (with the plugin directory being next to the plasmoid directory). However, if you can think of another name that makes its purpose more clear, please go ahead and change it :smile:

Reply:
Thanks!
I understand!